DOCS.md

```diff
@@ -241,6 +249,26 @@ Class | Method | HTTP request | Description
  - [GiantSwarmV4.V4ReleaseListItemChangelog](docs/V4ReleaseListItemChangelog.md)
  - [GiantSwarmV4.V4ReleaseListItemComponents](docs/V4ReleaseListItemComponents.md)
  - [GiantSwarmV4.V4UserListItem](docs/V4UserListItem.md)
+ - [GiantSwarmV4.V5AddClusterRequest](docs/V5AddClusterRequest.md)
```

Comment:
This is starting to look a bit weird, maybe we should remove the V4 from the GiantSwarmV4.

Reply:
Good point